app/assets/javascripts/discourse/app/controllers/users.js

```diff
@@ -47,13 +48,32 @@ export default Controller.extend({
       });
   },
 
+  loadGroups() {
+    this.store.findAll("group").then((groups) => {
+      const groupOptions = [];
+      groups.forEach((group) => {
```

Comment:
I wish I could just do 
```
this.store.findAll("group").then((groups) => {
  this.set("groupsOptions", groups);
})
```
But `groups` is an arrayProxy and can't be passed directly into a combo-box as options.